src/index.html

```diff
@@ -11,5 +11,29 @@
 </head>
 <body class="mat-typography mat-app-background">
   <app-root></app-root>
+  <!-- The core Firebase JS SDK is always required and must be listed first -->
+<script src="https://www.gstatic.com/firebasejs/8.0.0/firebase-app.js"></script>
+
+<!-- TODO: Add SDKs for Firebase products that you want to use
+     https://firebase.google.com/docs/web/setup#available-libraries -->
+<script src="https://www.gstatic.com/firebasejs/8.0.0/firebase-analytics.js"></script>
+
+<script>
+  // Your web app's Firebase configuration
+  // For Firebase JS SDK v7.20.0 and later, measurementId is optional
+  var firebaseConfig = {
+    apiKey: "AIzaSyCGGfJeR8w3OBgVPico3JySnAoESX6GpCk",
+    authDomain: "blockframes-samir.firebaseapp.com",
+    databaseURL: "https://blockframes-samir.firebaseio.com",
+    projectId: "blockframes-samir",
+    storageBucket: "blockframes-samir.appspot.com",
+    messagingSenderId: "550505593230",
+    appId: "1:550505593230:web:be698dda150d9033da8cdf",
+    measurementId: "G-H73BW4HWKL"
+  };
+  // Initialize Firebase
+  firebase.initializeApp(firebaseConfig);
+  firebase.analytics();
+</script>
```

Comment:
You don't need this if we initialize firebase in the module. Try to delete this.